src/ConnectedVMware/docs/New-AzConnectedVMwareVMGuestAgent.md

```diff
@@ -37,27 +37,43 @@ Create GuestAgent.
 
 ## EXAMPLES
 
-### Example 1: {{ Add title here }}
+### Example 1: Enable Guest Agent on VM Instances
 ```powershell
-{{ Add code here }}
+New-AzConnectedVMwareVMGuestAgent -ResourceUri "/subscriptions/204898ee-cd13-4332-b9d4-55ca5c25496d/resourceGroups/test-rg/providers/Microsoft.HybridCompute/machines/test-machine" -CredentialsUsername $VM_User_Name -CredentialsPassword $Secure_String_Pwd -ProvisioningAction "install"
```

Comment:
Can we customize this param name? We want to rename "ResoruceUri" to "MachineId"?